native/src/components/SimpleImage.tsx

```diff
@@ -8,7 +8,7 @@ import getCachedThumbnail from '../utils/getCachedThumbnail'
 import Icon from './base/Icon'
 
 const StyledImage = styled.Image<{ aspectRatio?: number }>`
-  ${props => props.aspectRatio && `aspect-ratio: ${props.aspectRatio};`}
+  ${props => props.aspectRatio !== undefined && `aspect-ratio: ${props.aspectRatio};`}
```

Comment:
In case anybody else wonders what happens with an aspect ratio of 0: The property is ignored.

Reply:
Hmm I didn't test this now but when I think to recall that when implementing the aspect ratio in the first place setting `aspectRatio` somehow broke stuff. But I might be wrong...